ssh/ssh.go

```diff
@@ -199,8 +199,24 @@ func writeSudoPassword(cmd *exec.Cmd, sudoPasswd string) (err error) {
 
 func (ctx *SSHContext) ActivateConfiguration(host Host, configuration string, action string) error {
 
+	var (
+		cmd *exec.Cmd
+		err error
+	)
+
+	cmd, err = ctx.Cmd(host, "test", "-f", "/.morph_disabled")
```

Comment:
Please move the check for whether a host is disabled to function on its own. The existing code is already messy enough as is ^^

Reply:
Sure, _cordon_ works fine for me. I used .morph_disabled as I remember the early version of puppet used /.puppet_disabled.

Reply:
Thank you, will do.